sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRepositoryAsyncClient.java

```diff
@@ -116,25 +146,40 @@ public String getRepository() {
     }
 
     /**
-     * Delete the repository.
+     * Delete the repository in the Azure Container Registry for the given {@link #getRepository() repository}.
      *
-     * @throws ClientAuthenticationException thrown if the client's credentials do not have access to modify the namespace.
-     * @throws ResourceNotFoundException thrown if the given name was not found.
-     * @return deleted repository properties.
+     * <p><strong>Code Samples</strong></p>
+     *
+     * <p>Delete the repository.</p>
+     *
+     * {@codesnippet com.azure.containers.containerregistry.async.repositoryclient.deleteRepository}
+     *
+     * @return It returns the count of the tags and artifacts that are deleted as part of the repository delete.
+     * @throws ClientAuthenticationException thrown if the client does not have access to the repository.
+     * @throws ResourceNotFoundException thrown if the repository with the given name was not found.
+     * @throws HttpResponseException thrown if any other unexpected exception is returned by the service.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<DeleteRepositoryResult> delete() {
         return this.deleteWithResponse().flatMap(FluxUtil::toMono);
     }
 
     /**
-     * Delete registry artifact.
+     * Deletes the registry artifact with the matching digest in the given {@link #getRepository() respository.}
+     * The delete operation supports delete of both child as well as a multi-arch artifact.
```

Comment:
Trying to parse this a bit.

The delete operation supports both deletion of children and multi-arch artifacts?

Reply:
artifact images are like a tree - with parent and chidren elements but docker tend to call multi-arch image.  I will actually go ahead and delete this line for now - fo I can see it being confusing.